package.json

```diff
@@ -9,7 +9,8 @@
     "dev": "NODE_ENV=development npm run start",
     "dev:search": "APP_NAME=search npm run dev",
     "dev:disco": "APP_NAME=disco npm run dev",
-    "test": "npm run version-check && npm run lint",
+    "test": "npm run version-check && npm run karma",
```

Comment:
Having eslint in the regular tests we advertise to contributors means they won't be surprised when tests pass locally but fail for style on travis (where we should definitely "run all the tests").

If eslint feels slow I'd recommend what we do in the linter, which is have subset test scripts (our code coverage is slow and messes with tracebacks on the linter, for instance).

So `npm test` should do everything, and `npm test-karma` or whatever can just do karma.